src/main/java/pl/gov/coi/pomocua/ads/dev/FakeUserRepository.java

```diff
@@ -0,0 +1,13 @@
+package pl.gov.coi.pomocua.ads.dev;
+
+import pl.gov.coi.pomocua.ads.UserId;
+import pl.gov.coi.pomocua.ads.users.User;
+import pl.gov.coi.pomocua.ads.users.UsersRepository;
+
+public class FakeUserRepository implements UsersRepository {
```

Comment:
A small thing, but... it's `Fake>User<Repository`, whether the interface is in plural: `UsersRepository`
Maybe let's keep it consistent? (either way)